File: Seção 13 - Linguagem Python/Exercícios/Exercicio_5.py
print("Digite dois numeros: ")
x = int(input())
y = int(input())

if x > y:
    z = y
    y = x
    x = z

soma = 0

for i in range(x + 1, y):
    if i % 2 != 0:
        soma = i + soma

print(f"SOMA DOS IMPARES = {soma}")

File: Seção 10 - Linguagem C++/Exercicios/Exercicio_5.cpp
#include <iostream>

using namespace std;

int main(){

    int n1, n2, nt, i, soma;

    cout << "Digite dois numeros: " << endl;
    cin >> n1;
    cin >> n2;

    if ( n1 > n2 ){
        nt = n2;
        n2 = n1;
        n1 = nt;
    }

    soma = 0;

    for(i = n1 + 1; i < n2 ; i++){
        if (i % 2 != 0){
            soma = soma + i;
        }
    }

    cout << "SOMA DOS IMPARES = " << soma;

    return 0;
    }


